source/renderer/app/components/wallet/wallet-import/WalletImportFileDialog.scss

```diff
@@ -29,6 +27,11 @@
   height: 100%;
   padding: 50px 80px;
 
+  @media (max-height: 600px) {
```

Comment:
@DeeJayElly why do we need these media queries? It should all work without them as it did before...

Reply:
Yeah since we previously had just Dialog and the container/content inside of it, it was implemented in a different way, and now we have Dialog which calls the ReactModal in background in order to render it (like on other screens, Wallet Restore, Wallet Create step screens), things are a bit different when it comes to nesting elements so containers are changed. 

Will try to move away from having a scrollbar outside main container and fitting it inside the container like it was before.

Reply:
Removed

Reply:
Fixed